fractor-doc-generator/src/RuleCodeSamplePrinter/FractorRuleCodeSamplePrinter.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace a9f\FractorDocGenerator\RuleCodeSamplePrinter;
+
+use a9f\Fractor\Application\Contract\FractorRule;
+use a9f\FractorDocGenerator\Printer\DiffCodeSamplePrinter;
+use Symplify\RuleDocGenerator\Contract\CodeSampleInterface;
+use Symplify\RuleDocGenerator\Contract\RuleCodeSamplePrinterInterface;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+
+final readonly class FractorRuleCodeSamplePrinter implements RuleCodeSamplePrinterInterface
+{
+    public function __construct(
+        private DiffCodeSamplePrinter $diffCodeSamplePrinter,
+    ) {
+    }
+
+    public function isMatch(string $class): bool
```

Comment:
```suggestion
    public function matches(string $class): bool
```

Reply:
That´s not possible because we do not owe the Interface.

Reply:
Ah, my bad, did not realize that.